custom_components/valetudo_vacuum_camera/common.py

```diff
@@ -66,8 +69,10 @@ def get_vacuum_mqtt_topic(vacuum_entity_id: str, hass: HomeAssistant) -> str | N
     Fetches the mqtt topic identifier from the MQTT integration. Returns None if it cannot be found.
     """
     try:
+        # Fix for Home Assistant 2026.6.0
+        DATA_MQTT: HassKey[MqttData] = HassKey("mqtt")
```

Comment:
@rohankapoorcom thanks well spotted will do ;)

Reply:
I'm saying that instead of using the string "mqtt", we should use the already existing MQTT_DOMAIN that we've imported on line 11.

The new line would look like this:

`DATA_MQTT: HassKey[MqttData] = HassKey(MQTT_DOMAIN)`